akvo/rest/views/project_editor.py

```diff
@@ -126,12 +129,16 @@ def log_changes(changes, user, project):
 def split_key(key):
     """
     Helper function for splitting the keys of the form data. Key input will be a string like
-    'rsr_relatedproject.relation.1234_new-0' and it will return a tuple as such:
-
-    ('rsr', 'relatedproject'), 'relation', ('1234', 'new-0')
+    'rsr_relatedproject.relation.1234_new-0' and it will return a KeyInfo namedtuple
     """
-    key_info = key.split('.')
-    return key_info[0].split('_'), key_info[1], key_info[2].split('_')
+    KeyParts = namedtuple('KeyParts', 'model, field, ids')
+
+    key_parts = KeyParts._make(key.split('.'))
```

Comment:
I would prefer to use `KeyParts(*key.split('.'))` instead. Do you have any thoughts/comments?

Reply:
Of course, much more elegant!